Readme.md

```diff
@@ -9,6 +9,7 @@
 * Renamed the `DateTime` CSL type to `Instant`.
 * Added support for new date/time types.
 * Added description and termination time as parameters to termination command
+* Changed `declarationExpressionArguments` to `declarationExpressionArgument. It is now a single value instead of a list of values.
```

Comment:
```suggestion
* Changed `declarationExpressionArguments` to `declarationExpressionArgument`. It is now a single value instead of a list of values.
```